### PR TITLE
Redesign: border width panel

### LINKED ISSUE #6475: Redesign: Update border width panel
## Feature Description
See [Figma](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=3471%3A7322)

- [x] Use inputs from design system
- [x] Use toggle from design system
- [x] Use color from design system
- [x] Use tooltip from design system

## Additional Context
Screenshot 2021-02-22 at 13 03 48

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by miina**
## Summary
Redesigns the border panel.

## User-facing changes

| Before | After |
|--------|------|
| Screenshot 2021-03-05 at 19 12 19 | Screenshot 2021-03-05 at 19 43 28 |
| Screenshot 2021-03-05 at 19 12 25 | Screenshot 2021-03-05 at 19 11 02 | 
### QA

- [ ] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

[Figma](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=4647%3A0)

1. Add a rectangle shape
2. Verify the border panel looks like in Figma
3. Unlock the border width to see all the inputs
4. Add some values to see the color input
5. Verify that all looks as expected.

### UAT

- [x] UAT should use the same steps as above.

This PR can be tested by following these steps:

1.

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testiing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [ ] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #6475

### CODE REVIEW

**Comment by barklund** [comment]
@googlebot I consent.

File: assets/src/design-system/components/input/index.js [hunk 1]
```diff
@@ -90,10 +90,14 @@ const InputContainer = styled.div(
...
-  ({ theme }) => css`
+  ({ hasSuffix, theme }) => css`
     height: 100%;
     width: 100%;
-    padding: 0 8px 0 0;
+    padding: 0;
+    ${hasSuffix &&
+    css`
+      padding-right: 8px;
+    `}
```
Comment (barklund) [inline assets/src/design-system/components/input/index.js:100]: @samwhale, added this here, so there's no unnecessary padding if no suffix following the input. Just FYI.

File: assets/src/design-system/components/input/index.js [hunk 2]
```diff
@@ -194,9 +199,10 @@ export const Input = forwardRef(
             onBlur={handleBlur}
             onFocus={handleFocus}
             value={displayedValue}
+            hasSuffix={hasSuffix}
             {...props}
           />
-          {suffix && (
+          {hasSuffix && (
```
Comment (barklund) [inline assets/src/design-system/components/input/index.js:205]: `suffix=0` would actually have printed `0` to DOM before, as it's falsey but printable.

File: assets/src/edit-story/components/panels/design/border/borderWidth.js [hunk 3]
```diff
@@ -20,43 +20,47 @@
...
-  padding: 6px 6px;
-  border-radius: 4px;
-  max-width: ${({ locked }) => (locked ? 'initial' : '36px')};
-`;
-
-const Space = styled.div`
-  flex: 0 0 10px;
+const Separator = styled.div`
+  width: 8px;
+  margin: -20px 0 6px;
```
Comment (barklund) [inline assets/src/edit-story/components/panels/design/border/borderWidth.js:48]: @miina, I removed all the spacing and left that to the parent flex element's `space-between`. Keeps all components at the proper size regardless of right scrollbar in panel:

| Without scrollbar | With scrollbar |
|-|-|
| Screen Shot 2021-03-06 at 18 32 29 | Screen Shot 2021-03-06 at 18 32 11 |

File: assets/src/edit-story/components/panels/design/border/borderWidth.js [hunk 4]
```diff
@@ -65,9 +69,31 @@ const LabelText = styled.span`
...
 
 const ToggleWrapper = styled.div`
-  height: ${({ locked }) => !locked && '60px'};
+  padding-top: 2px;
+  ${({ locked }) =>
+    locked &&
+    css`
+      padding-left: 8px;
+    `};
+  align-self: stretch;
```
Comment (barklund) [inline assets/src/edit-story/components/panels/design/border/borderWidth.js:78]: @miina, cleaned this up a bit too. The old syntax relied on `height: false` being removed from the output (which it is, but still a bit hacky).

**Review by barklund — APPROVED** [review]

File: assets/src/edit-story/components/panels/design/border/borderWidth.js (hunk 4 above)
Comment (miina) [reply assets/src/edit-story/components/panels/design/border/borderWidth.js:78]: Thank you!

**Comment by csossi** [comment]
Verified in QA

**Pull request merged**